<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageoffres</class>
 <widget class="QDialog" name="pageoffres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1829</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>-30</y>
     <width>1861</width>
     <height>1011</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for QRadioButton */
QRadioButton {
    color: white; /* Text color for radio buttons */
    font-size: 16px; /* Font size */
    font-weight: bold; /* Bold text */
    margin: 5px; /* Margin around radio buttons */
}

/* Style for QLabel (text label) */
QLabel {
    color: black; /* Text color for labels */
    font-size: 16px; /* Font size */
    font-weight: bold; /* Bold text */
    margin: 5px; /* Margin around labels */
}

/* Style for QPushButton */
QPushButton {
    background-color: #3498db; /* Blue color for buttons */
    border: 1px solid #2980b9; /* Darker blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding inside buttons */
    color: white; /* Text color on buttons */
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #2980b9; /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #21618c; /* Even darker blue when pressed */
}

/* Style for QTabWidget */
QTabWidget {
    color: white; /* Text color for tab widget */
    font: 10pt &quot;Bebas Neue&quot;; /* Font style and size */
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>0</y>
       <width>1861</width>
       <height>891</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
/* Style for QGroupBox */
QRadioButton{ color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}
QComboBox {
    background-color: #2980b9; /* Darker blue color for the combobox */
    border: 1px solid #1c669a; /* Darker border color */
    border-radius: 5px; /* Border radius for the combobox */
    padding: 5px; /* Padding inside the combobox */
    color: black; /* Text color inside the combobox */
}
QLineEdit {
    background-color: #2980b9; /* Darker blue color for input boxes */
    border: 1px solid #1c669a; /* Even darker border color */
    border-radius: 5px; /* Border radius for input boxes */
    padding: 5px; /* Padding for input boxes */
    color: black; /* Text color inside input boxes */
}

/* Style for QLabel (text label) */
QLabel {
    color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}

/* Style for QPushButton */
QPushButton {
    background-color: #3498db; /* Blue color for buttons */
    border: 1px solid #2980b9; /* Border color for buttons */
    border-radius: 5px; /* Border radius for buttons */
    padding: 5px 10px; /* Adjust padding as needed */
    color: white; /* Text color on buttons */
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #2980b9; /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #21618c; /* Even darker blue when pressed */
}

/* Add more styles as needed for your specific application */
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="id_offre">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Numbers only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>610</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>statut</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>id offre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>149</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>titre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titre_o">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>Characters only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>localisation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="localisation_o">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>470</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>510</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>date pub</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_pub_o">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>570</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>YYYY-MM-DD</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_offres">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>120</y>
        <width>881</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 15pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>description</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="description_o">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color: black;
font: 22pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>titre</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 22pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>localisation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortAscButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2980b9; /* Darker blue background color */
    border: 1px solid #1c669a; /* Darker border color */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 15px; /* Padding for the button */
    color: white; /* Text color */
    font-weight: bold; /* Bold text */
}

QPushButton:hover {
    background-color: #3498db; /* Slightly lighter blue when hovered */
}

QPushButton:pressed {
    background-color: #1c669a; /* Even darker blue when pressed */
}

QPushButton:disabled {
    background-color: #bdc3c7; /* Gray color when disabled */
    color: #7f8c8d; /* Gray text when disabled */
}</string>
      </property>
      <property name="text">
       <string>ASCendant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortDescButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2980b9; /* Darker blue background color */
    border: 1px solid #1c669a; /* Darker border color */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 15px; /* Padding for the button */
    color: white; /* Text color */
    font-weight: bold; /* Bold text */
}

QPushButton:hover {
    background-color: #3498db; /* Slightly lighter blue when hovered */
}

QPushButton:pressed {
    background-color: #1c669a; /* Even darker blue when pressed */
}

QPushButton:disabled {
    background-color: #bdc3c7; /* Gray color when disabled */
    color: #7f8c8d; /* Gray text when disabled */
}</string>
      </property>
      <property name="text">
       <string>DESCendant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1501</width>
        <height>921</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/sports-nautiques.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>-20</y>
        <width>2081</width>
        <height>991</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/qz.jfif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>760</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/google-plus_9723940.png);
background-color: transparent; border: none;color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>760</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/3597075.png);
background-color: transparent; border: none;color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>760</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/delete_10257834.png);
background-color: transparent; border: none;color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>20</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f0f0f0;
border: 5px solid #ccc;
padding: 5px;
border: 1px solid #0078d7;
font: 12pt &quot;Bebas Neue&quot;; 
border: 1px solid #ccc;
border-color: black;
padding: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refrech_pb">
      <property name="geometry">
       <rect>
        <x>1500</x>
        <y>10</y>
        <width>101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color: white;
border-image: url(:/img/img/load.png);
font: 22pt &quot;Bebas Neue&quot;; text-decoration: underline;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>1430</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color: white;
font: 22pt &quot;Bebas Neue&quot;; text-decoration: underline;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/img/img/search.png</normaloff>:/img/img/search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; border: none;color:  black;
font: 22pt &quot;Bebas Neue&quot;;</string>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QWidget" name="lab_chart_pie" native="true">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>170</y>
        <width>591</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="locaasc">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>80</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2980b9; /* Darker blue background color */
    border: 1px solid #1c669a; /* Darker border color */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 15px; /* Padding for the button */
    color: white; /* Text color */
    font-weight: bold; /* Bold text */
}

QPushButton:hover {
    background-color: #3498db; /* Slightly lighter blue when hovered */
}

QPushButton:pressed {
    background-color: #1c669a; /* Even darker blue when pressed */
}

QPushButton:disabled {
    background-color: #bdc3c7; /* Gray color when disabled */
    color: #7f8c8d; /* Gray text when disabled */
}</string>
      </property>
      <property name="text">
       <string>asc localisation</string>
      </property>
     </widget>
     <widget class="QComboBox" name="statut_o">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>670</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Actif</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non-Actif</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>30</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2980b9; /* Darker blue background color */
    border: 1px solid #1c669a; /* Darker border color */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 15px; /* Padding for the button */
    color: white; /* Text color */
    font-weight: bold; /* Bold text */
}

QPushButton:hover {
    background-color: #3498db; /* Slightly lighter blue when hovered */
}

QPushButton:pressed {
    background-color: #1c669a; /* Even darker blue when pressed */
}

QPushButton:disabled {
    background-color: #bdc3c7; /* Gray color when disabled */
    color: #7f8c8d; /* Gray text when disabled */
}</string>
      </property>
      <property name="text">
       <string>afficher tableau</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <zorder>lab_chart_pie</zorder>
     <zorder>label_12</zorder>
     <zorder>label_11</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>titre_o</zorder>
     <zorder>label_6</zorder>
     <zorder>localisation_o</zorder>
     <zorder>id_offre</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>date_pub_o</zorder>
     <zorder>tab_offres</zorder>
     <zorder>label_21</zorder>
     <zorder>description_o</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>sortAscButton</zorder>
     <zorder>sortDescButton</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>pb_modifier</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>lineEdit</zorder>
     <zorder>refrech_pb</zorder>
     <zorder>search</zorder>
     <zorder>radioButton_3</zorder>
     <zorder>locaasc</zorder>
     <zorder>statut_o</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-40</y>
     <width>1851</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/R (1).jfif);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="condpb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bebas Neue</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/6449076.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="historique">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18pt &quot;Bebas Neue&quot;; 
border: 1px solid #ccc;
border-color: black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Historique</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>630</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18pt &quot;Bebas Neue&quot;; 
border: 1px solid #ccc;
border-color: black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Statut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pdf">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18pt &quot;Bebas Neue&quot;; 
border: 1px solid #ccc;
border-color: black;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>tabWidget</zorder>
  <zorder>condpb</zorder>
  <zorder>stat</zorder>
  <zorder>historique</zorder>
  <zorder>pb_pdf</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
